app/controllers/items_controller.rb

```diff
@@ -5,12 +5,30 @@ class ItemsController < ApplicationController
   before_action :find_item, only: %i[show toggle_favorite]
 
   def index
+    # Gets current user coordinates from the postal code the user entered
+    # Currently using static postal code, we can update this when we fix the postal code issue<<<<<<<<<<<<<<<<
+    @current_postal_code = nil #params[:postal_code]
```

Comment:
You can enter a custom fixed postal code to test, the reason it's nil is because of line 19 in the items_controller.